### PR TITLE
run spring IT on spring boot 2.4.x

### PR DESCRIPTION
**Pull request by yiliuTo**

### CODE REVIEW

File: sdk/spring/spring-test-template.yml [hunk 1]
```diff
@@ -115,3 +115,44 @@ stages:
...
+              goals: 'install'
+              options: '$(DefaultOptions) -DskipTests -Dgpg.skip -Dmaven.javadoc.skip=true -Dcheckstyle.skip=true -Dspotbugs.skip=true -Drevapi.skip=true -pl $(ProjectList) -am'
+              mavenOptions: '$(MemoryOptions) $(LoggingOptions)'
+              javaHomeOption: 'JDKVersion'
+              jdkVersionOption: $(JavaBuildVersion)
+              jdkArchitectureOption: 'x64'
+              publishJUnitResults: false
+
+          - script: |
+                az cloud set --name AzureChinaCloud
```
Comment (yiliuTo) [inline sdk/spring/spring-test-template.yml:137]: For there are scirpts being scheduled twice, is it valuable that we move each of them to a file and call it then? But each scirpt only contains 2-3 lines of code....

**Review by chenrujun — APPROVED** [review]
LGTM.

**Pull request merged**